### ISSUE TITLE
Renderers in home directory

### ISSUE THREAD

**Issue by Techmeology** [UniversalMediaServer/UniversalMediaServer#1096]
Hello

I am trying to set up Universal Media Server with a custom renderer. I have UMS set up to run as its own user. It correctly loads the configuration file from `/home/ums/.config/UMS/UMS.conf`, but it insists on using the renderers in `/opt/ums/renderers` even though it finds the ones in `/home/ums/.config/UMS/renderers`.

As a test, I moved `/opt/ums/renderers` to `/home/ums/.config/UMS/`. The `debug.log` contained:
`DEBUG 2016-11-14 03:58:07.694 [main] Added file watch at /home/ums/.config/UMS/renderers: /home/ums/.config/UMS/renderers/AnyCast.conf`
and
```DEBUG 2016-11-14 03:58:10.046 [New I/O worker #1] Marking renderer "Unknown renderer" at /192.168.1.4 as unrecognized
INFO  2016-11-14 03:58:10.046 [New I/O worker #1] Media renderer was not recognized. Possible identifying HTTP headers:User-Agent: Android/4.4.4 UPnP/1.0 Cling/2.0```

But when I moved the renderers back to their original location:
```DEBUG 2016-11-14 04:08:48.531 [main] Added file watch at renderers: renderers/AnyCast.conf
DEBUG 2016-11-14 04:08:48.531 [main] Ignored AnyCast configuration```
and
`DEBUG 2016-11-14 04:08:50.811 [New I/O worker #2] Matched media renderer "Android" based on headers {Host=192.168.1.3:5001, User-Agent=Android/4.4.4 UPnP/1.0 Cling/2.0}`

It appears as if, although it *finds* the renderers in the location I put them in, it doesn't actually *use* them.

Is this a bug, or am I missing something?

Thanks

**Comment by Nadahar** [comment]
As far as I know renderer configurations found in ```.config/UMS/renderers``` should have priority over those placed in ```/opt/ums/renderers```. I'm not sure what is happening or why, but a trace log might help shed some light on this.

If you're running the GUI please follow [this procedure](http://www.universalmediaserver.com/forum/viewtopic.php?f=9&t=556). If you're running headless, start UMS with "trace" as argument:
```./UMS.sh trace``` and post ```debug.log``` and ```UMS.conf``` after trying to connect the renderer.

**Comment by atamariya** [comment]
@Techmeology Renderer conf. is not designed to be user configurable unlike other config. parameters. You should edit the conf. in /opt/ums/renderers for any changes that you desire.

**Comment by Nadahar** [comment]
@atamariya I disagree, unless I'm completely misinformed user overrides are supposed to be placed in ```<profile folder>/renderers``` which would translate to ```/home/ums/.config/UMS/renderers``` given that the user name is ```ums```.
